sql/sp.cc

```diff
@@ -2571,13 +2571,20 @@ uint sp_get_flags_for_command(LEX *lex)
   case SQLCOM_RESET:
     flags= sp_head::HAS_SQLCOM_RESET;
     break;
+  /*
+    ALTER TABLE may return a result set (in case of CHECK PARTITION
+    at least), but doesn't have to. See SQLCOM_EXECUTE comment why
```

Comment:
Would it be possible to explain "the why" here?

Reply:
Done